tensorpack/tfutils/symbolic_functions.py

```diff
@@ -301,22 +292,23 @@ def triplet_loss(anchor, positive, negative, margin, extra=False):
     `FaceNet: A Unified Embedding for Face Recognition and Clustering <https://arxiv.org/abs/1503.03832>`
     by Schroff et al.
 
-    Learn embeddings from an anchor point and a similar input (p) as well as a not similar input (n)
-    Intuitively, a matching-pair (anchor, positive) should have a smaller relative distance
+    Learn embeddings from an anchor point and a similar input (positive) as
+    well as a not-similar input (negative).
+    Intuitively, a matching pair (anchor, positive) should have a smaller relative distance
     than a non-matching pair (anchor, negative).
 
     .. math::
-        \min \max(0, m + \Vert a-p\Vert^2 - \Vert a-n\Vert^2)
+        \max(0, m + \Vert a-p\Vert^2 - \Vert a-n\Vert^2)
```

Comment:
You can. They look the same to me. maybe will have some very slightly performance difference?